### DISCUSSION TITLE
Is the relationship manager's data renderable on a separate page?

### DISCUSSION THREAD

**Discussion by aimonext** [filamentphp/filament#8534] [Help]
I'm utilizing a Relationship Manager that assists in performing operations while on the '/view' or '/edit' route. However, I'm exploring the possibility of separating these functionalities. For instance, I have a sales invoice containing cart items, and currently, I display the cart items list on the '/view' or '/edit' page. I'm interested in segregating this functionality to a dedicated '/cart' page. Is such separation feasible?

**Comment by leandrocfe** [comment]
*Relation managers are interactive tables that allow administrators to list, create, attach, associate, edit, detach, dissociate and delete related records without leaving the resource's*
But you are free to create custom pages and custom resources to achieve this.
